packages/bridge/src/runtime/composables/ssr.ts

```diff
@@ -14,5 +14,11 @@ export function useRequestHeaders (include?: any[]) {
 }
 
 export function useRequestEvent (nuxtApp: NuxtAppCompat = useNuxtApp()): H3Event {
-  return nuxtApp.ssrContext?.event as H3Event
+  // check if H3 event is available
+  if (nuxtApp.ssrContext?.event) { return nuxtApp.ssrContext.event }
+  // Check if we created H3 event manually before
+  if (nuxtApp.ssrContext?._event) { return nuxtApp.ssrContext._event }
+  // Create H3 event https://github.com/nuxt/bridge/pull/999#discussion_r1413049422
+  nuxtApp.ssrContext._event = createEvent(nuxtApp.ssrContext?.req, nuxtApp.ssrContext?.res)
```

Comment:
Mind you this is a very fragile low-level method and behavior can be broken in future releases but considering that it is acceptable for edge-case usage in bridge, looks good workaround...